src/applications/facility-locator/utils/helpers.jsx

```diff
@@ -178,6 +178,11 @@ export const formatOperatingHours = operatingHours => {
     return operatingHours;
   }
 
+  // If there are multiple time ranges, such as for lunch hours
+  if (split(sanitizedOperatingHours, '-').length >= 3) {
+    return operatingHours;
+  }
```

Comment:
*SHOULD*

This logic looks good 👍  

Later on, though, `split(sanitizedOperatingHours)` is being called again. I think we could probably avoid doing this twice. It might make sense as part of that to move this block closer to the other one that deals with this split data.